public/controllers/management/components/management/configuration/log-collection/log-collection-sockets.js

```diff
@@ -57,27 +58,27 @@ class WzConfigurationLogCollectionSockets extends Component {
   }
   render() {
     const { currentConfig } = this.props;
-    const items = isArray(currentConfig['logcollector-socket'].target)
-      ? settingsListBuilder(currentConfig['logcollector-socket'].target, 'name')
+    const items = isArray(currentConfig[LOGCOLLECTOR_SOCKET_PROP].target)
+      ? settingsListBuilder(currentConfig[LOGCOLLECTOR_SOCKET_PROP].target, 'name')
       : [];
     return (
       <Fragment>
-        {currentConfig['logcollector-socket'] &&
-          isString(currentConfig['logcollector-socket']) && (
+        {currentConfig[LOGCOLLECTOR_SOCKET_PROP] &&
+          isString(currentConfig[LOGCOLLECTOR_SOCKET_PROP]) && (
             <WzNoConfig
-              error={currentConfig['logcollector-socket']}
+              error={currentConfig[LOGCOLLECTOR_SOCKET_PROP]}
               help={helpLinks}
             />
           )}
-        {currentConfig['logcollector-socket'] &&
-        !isString(currentConfig['logcollector-socket']) &&
-        !currentConfig['logcollector-socket'].target ? (
+        {currentConfig[LOGCOLLECTOR_SOCKET_PROP] &&
+        !isString(currentConfig[LOGCOLLECTOR_SOCKET_PROP]) &&
+        !currentConfig[LOGCOLLECTOR_SOCKET_PROP].target ? (
           <WzNoConfig error="not-present" help={helpLinks} />
         ) : null}
-        {currentConfig['logcollector-socket'] &&
-        !isString(currentConfig['logcollector-socket']) &&
-        currentConfig['logcollector-socket'].target &&
-        currentConfig['logcollector-socket'].target.length ? (
+        {currentConfig[LOGCOLLECTOR_SOCKET_PROP] &&
+        !isString(currentConfig[LOGCOLLECTOR_SOCKET_PROP]) &&
```

Comment:
This kind of conditional can be replaced by the optional chaining operator. ?.
It would simplify the conditional and improve readability.
e.g.
`currentConfig?.[LOGCOLLECTOR_SOCKET_PROP]?.target?.length`